update/update-version.md

```diff
@@ -86,25 +86,44 @@ If you have both the code and the Terraform state file stored locally on your ma
     - jq
 
 1. Set the `IBMCLOUD_API_KEY` variable as an environment variable.
-    1. IBMCLOUD_API_KEY : This is an IBM Cloud API key, which can be used for accessing the Projects/Schematics Workspace. For more information, see https://cloud.ibm.com/docs/account?topic=account-userapikey&interface=ui
+    1. IBMCLOUD_API_KEY : This is an IBM Cloud API key, which can be used for accessing the VPC. For more information, see https://cloud.ibm.com/docs/account?topic=account-userapikey&interface=ui
     ```sh
-        export IBMCLOUD_API_KEY="<API-KEY>" #pragma: allowlist secret
+    export IBMCLOUD_API_KEY="<API-KEY>" #pragma: allowlist secret
     ```
 
-1. Retrive the VPC IDs of the VPC on which all the VSIs are deployed, including the region of VPC.
-    1. VPC ID can be retrieved from the terraform output by running `terraform output`.
+1. Retrive the VPC ID of the VPC on which all the VSIs are deployed, including the region of VPC.
+    1. VPC ID can be retrieved from the terraform output by running `terraform output` or use the following command to get the VPC ID
+    ```sh
+    terraform output -json | jq -r '.. | objects | .value' | jq -r '.. | objects | select(.vpc_id != null) | .vpc_id' | sort -u
+    ```
+
+1. Download/Copy the script to the same directory in which the state file exists.
+```sh
+curl https://raw.githubusercontent.com/terraform-ibm-modules/terraform-ibm-landing-zone-vsi/refactor/update/update_v3.x.x_to_v4.x.x.sh > update_v3.x.x_to_v4.x.x.sh
```

Comment:
you will wan't this to point to main branch for this doc (I know it doesn't exist yet, but it will when PR is merged)